src/plugins/intel_cpu/src/config.cpp

```diff
@@ -271,11 +271,6 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
         streamExecutorConfig._streams_changed = true;
     }
 
-#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
-    // TODO: multi-stream execution has functional issues on ARM target
-    streamExecutorConfig._streams = 1;
```

Comment:
Let's keep this WA before https://github.com/openvinotoolkit/openvino/pull/17445 merged

Reply:
Ok, done.